docs/track.rst

```diff
@@ -194,8 +201,8 @@ Example::
         }
     ]
 
-templates
-.........
+templates (deprecated)
```

Comment:
In which context are they deprecated? Do you mean in Elasticsearch or in Rally? If it's the former, I don't think we should document so explicitly what is deprecated in a certain Elasticsearch version given the wide range of major versions.

Reply:
yes i was thinking we might remove support but given our discussions re ES support for 2 majors +1.